linux_os/guide/services/ssh/ssh_server/sshd_use_priv_separation/oval/shared.xml

```diff
@@ -36,7 +47,11 @@ root privileges when not needed.</description>
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sshd_use_priv_separation" version="1">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
+    {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+    <ind:pattern operation="pattern match">^[\s]*(?i)UsePrivilegeSeparation(?-i)[\s]+yes[\s]*(?:|(?:#.*))?$</ind:pattern>
+    {{% else %}}
```

Comment:
Instead of this, please create a variable that is used to select either `yes` or `sandbox`. The default should be `sandbox` as it is more restrictive. Then, select `yes` for your profile. Variable should be called `var_sshd_priv_separation`